tests/UnifiedSpecTests/UnifiedTestRunner.php

```diff
@@ -339,6 +289,52 @@ private function getServerVersion() : string
         throw new UnexpectedValueException('Could not determine server version');
     }
 
+    /**
+     * Return the topology type.
+     *
+     * @throws UnexpectedValueException if topology is neither single nor RS nor sharded
+     */
+    private function getTopology() : string
+    {
+        // TODO: detect load-balanced topologies once PHPLIB-671 is implemented
```

Comment:
IIRC, this will entail checking an environment variable and likely preempt this `switch`.